src/libraries/System.Text.Json/tests/Serialization/ExtensionDataTests.cs

```diff
@@ -1077,8 +1077,7 @@ public override object Read(ref Utf8JsonReader reader, Type typeToConvert, JsonS
 
             public override void Write(Utf8JsonWriter writer, object value, JsonSerializerOptions options)
             {
-                // Since we are converter for object, the string converter will be called instead of this.
-                throw new InvalidOperationException();
+                writer.WriteStringValue((string)value);
```

Comment:
Pls add comment: 

`// Since we are in a user-provided (not internal to S.T.Json) object converter, this converter will be called, not the string converter.`